css/style.css

```diff
@@ -0,0 +1,316 @@
+
+nav {
+  font-weight: 400;
+}
+
+@media (max-width: 650px) {
+  nav {
+    margin-top: 10px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    padding: 0 50px;
+  }
+  nav li {
+    padding-bottom: 5px;
+  }
+}
+
+nav > ul {
+  width: 35vw;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+@media (max-width: 650px) {
+  nav > ul {
+    flex-direction: column;
+  }
+}
+
+
+#hero {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  height: 200px;
+  margin-top: 50px;
+}
+
+#hero > h2 {
+  margin-bottom: 20px;
+  word-wrap: break-word;
+}
+
+#hero input[type='email'] {
+  max-width: 275px;
+  width: 100%;
+  padding: 5px;
+}
+
+#hero input[type='submit']:hover {
+  background-color: orange;
+  transition:  1s;
+}
+#hero input[type='submit'] {
+  width: 150px;
+  margin-top: 10px;
+  padding: 5px;
+  background-color: #f1c40f;
+  border: 0;
+  font-size: larger;
+  font-weight: bolder;
+}
+
+@media (max-width: 650px) {
+  #hero {
+    margin-top: 120px;
+  }
+}
+
+#features {
+  margin-top: 30px;
+
+}
+.list{
+  display: flex;
+}
+#features .icon {
+  display: flex;
+  font-size: 50px;
+  align-items: center;
+  justify-content: center;
+  height: 125px;
+  width: 20vw;
+  color: darkorange;
+}
+
+@media (max-width: 550px) {
+  #features .icon {
+    display: none;
+  }
+}
+
+#features .desc {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 125px;
+  width: 80vw;
+  padding: 5px;
+}
+
+@media (max-width: 550px) {
+  #features .desc {
+    width: 100%;
+    text-align: center;
+    padding: 0;
+    height: 150px;
+  }
+}
+
+@media (max-width: 650px) {
+  #features {
+    margin-top: 0;
+  }
+}
+
+#how-it-works {
+  margin-top: 50px;
+  display: flex;
+  justify-content: center;
+}
+
+#how-it-works > iframe {
+  max-width: 560px;
+  width: 100%;
+}
+
+#pricing {
+  margin-top: 60px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  width: calc(100% / 3);
+  margin: 10px;
+  border: 1px solid #000;
+  border-radius: 3px;
+}
+
+.card > .level {
+  background-color: #ddd;
+  color: black;
+  padding: 15px 0;
+  width: 100%;
+  text-transform: uppercase;
+  font-weight: 700;
+}
+
+.card > h2 {
+  margin-top: 15px;
+}
+
+.card > h2 {
+  margin: 15px 0;
+}
+
+.card > ol {
+  margin: 15px 0;
+}
+
+.card > ol > li {
+  padding: 5px 0;
+}
+
+.card > button {
+  border: 0;
+  margin: 15px 0;
+  background-color: #f1c40f;
+  font-weight: 400;
+}
+
+.card > button:hover {
+  background-color: orange;
+  transition:  1s;
+}
+
+@media (max-width: 800px) {
+  #pricing {
+    flex-direction: column;
+}
+ footer {
+  width: 100%;
+ }
+.card {
+  max-width: 300px;
+  width: 100%;
+  margin: 0 auto;
+  margin-bottom: 10px;
+ }
+
+
```

Comment:
Remove excess spacing.

Reply:
- [ ] Just as I suggested above, try to make your review comments a suggestion instead of an order. Also, try to give reasons for your suggested actions or changes. :+1: :small_airplane: 
- [OPTIONAL] I'll suggest you also try to use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.